docs/self-managed/platform-deployment/helm-kubernetes/guides/running-custom-connectors.md

```diff
@@ -0,0 +1,59 @@
+---
+id: running-custom-connectors
+title: "Running custom Connectors"
+description: "Run custom Connectors in your Helm Kubernetes cluster."
+---
+
+You can deploy your custom **Connector** in your Helm Kubernetes cluster along with Connectors Bundle.
```

Comment:
I need to add Connectors Bundle to the glossary to determine casing, an FYI to myself 👍